tests/unit/multi.tcl

```diff
@@ -657,7 +657,7 @@ start_server {tags {"multi"}} {
         # check that GET and PING are disallowed on stale replica, even if the replica becomes stale only after queuing.
         r multi
         r get xx
-        $r1 replicaof localhsot 0
+        $r1 replicaof localhost_ 0
```

Comment:
why the `_`?

Reply:
You're right, `localhsot` is an intentional typo of `localhost`. Unlike the `filx`->`file` change, I think we can keep this one as `localhost_` unless someone else has any objections.